frontend/src/ErrorHandlerPage/index.jsx

```diff
@@ -0,0 +1,92 @@
+import React, { useState } from 'react';
+import {
+  View, Text, TouchableOpacity, StyleSheet, Image,
+  SafeAreaView, TouchableWithoutFeedback,
+} from 'react-native';
+import PropTypes from 'prop-types';
+
+import Gradient from 'react-native-css-gradient';
+import Popup from '../common/components/Popup';
+import ModalHeader from '../common/components/ModalHeader';
+import CloseButton from './CloseButton';
+
+import error from '../../assets/error.gif';
+import commonColors from '../common/colors';
+
+const styles = StyleSheet.create({
+  background: {
+    flex: 1,
+    backgroundColor: 'rgba(0,0,0,0.6)',
+    justifyContent: 'center',
+    alignItems: 'center',
+  },
+  container: {
+    width: 300,
+    backgroundColor: 'rgb(202,202,202)',
+    borderWidth: 2,
+    borderColor: 'yellow',
+    borderRadius: 6,
+  },
+  mainBody: {
+    padding: 20,
+    alignItems: 'center',
+  },
+  modalHeader: {
+    backgroundColor: '#FF3333',
+    borderTopStartRadius: 0,
+    borderTopRightRadius: 0,
+  },
+  imgStyle: {
+    height: 175,
+    width: 175,
+    borderRadius: 6,
+  },
+  textStyle: {
+    margin: 10,
+    fontSize: 20,
+    color: commonColors.whiteColor,
+    fontWeight: 'bold',
+  },
+});
+
+
+function ErrorHandlerPage({ ErrorInfo }) {
```

Comment:
```suggestion
function ErrorPopup({ title, message }) {
```
I think component can be started with upper case,
and sometimes we may need to custom error's title.